### PR TITLE
Ironrdp manual docs

### PR DESCRIPTION
**Pull request by ibeckermayer**
Updates the manual documentation for use with IronRDP, namely adding instructions to enable RemoteFX on the Teleport Access Policy GPO. **Best reviewed commit by commit.**

image

### CODE REVIEW

File: docs/pages/desktop-access/active-directory-manual.mdx [hunk 1]
```diff
@@ -385,31 +401,61 @@ policies that allow remote connections to domain computers.
    ![Open the Firewall](../../img/desktop-access/firewall.png)
    </Figure>
 
-1. Open PowerShell and run the following command to update your Teleport
-   group policy object:
-   
+1. To ensure your GPO update takes effect immediately on this host,
+   Open PowerShell and run the following command (optional):
```
Comment (ptgott) [inline docs/pages/desktop-access/active-directory-manual.mdx:405]:
```suggestion
   open PowerShell and run the following command (optional):
```

File: docs/pages/desktop-access/active-directory-manual.mdx [hunk 2]
```diff
@@ -457,25 +503,23 @@ To update the Teleport group policy object to use the new certificate template:
...
+1. Expand **`Computer Configuration > Policies > Windows Settings > Public Key Policies`**.
 
 1. Double-click **Certificate Services Client - Auto-Enrollment**, then select
    **Enabled** in the Configuration Model.
 
-1. Open PowerShell and run the following command to update your Teleport
-   group policy object:
-   
+1. To ensure your GPO update takes effect immediately on this host,
+   Open PowerShell and run the following command (optional):
```
Comment (ptgott) [inline docs/pages/desktop-access/active-directory-manual.mdx:521]:
```suggestion
   open PowerShell and run the following command (optional):
```

**Review by ptgott — APPROVED** [review]

**Comment by ptgott** [comment]
Double-checking whether we should backport this one

**Comment by zmb3** [comment]
No backport. v15+ only.

**Review by probakowski — APPROVED** [review]

**Pull request merged**